src/libraries/Microsoft.PowerFx.Core/Lexer/Tokens/IdentToken.cs

```diff
@@ -70,7 +71,18 @@ internal override Token Clone(Span ts)
         // REVIEW ragru: having a property for every possible error isn't scalable.
         internal bool HasDelimiters => HasDelimiterStart;
 
-        internal bool HasErrors => IsModified || (HasDelimiterStart && !HasDelimiterEnd);
+        /// <summary>
+        /// Whether an identifier has errors.
+        /// </summary>
+        public bool HasErrors => IsModified || (HasDelimiterStart && !HasDelimiterEnd);
+
+        // TODO: How to do this properly?
```

Comment:
This is probably not good. Looking forward for refactor suggestions.

Reply:
This seems reasonable place.